build_runner/lib/src/generate/phase.dart

```diff
@@ -31,8 +31,6 @@ abstract class BuildAction {
   /// [excludes]. Glob syntax is supported for both [inputs] and [excludes].
   factory BuildAction(Builder builder, String package,
       {List<String> inputs, List<String> excludes, bool isOptional}) {
-    inputs ??= const ['**'];
```

Comment:
we still need this default here since its passed to a non-optional arg not forwarded to an optional one